docs/digger/installation/installer-variables.asciidoc

```diff
@@ -56,12 +56,21 @@ you must specify OAuth token so adding the key automatically doesn't fail
 |Number of max number of concurrent Android builds that are allowed.
 |5
 |No
-|===
 
 |jenkins_private_key_password
 |Jenkins SSH private key password (if password protected)
 |None
 |No
+
+|jenkins_private_key_path
+|Jenkins SSH private key path
+|~/.ssh/id_rsa
+|No
+
+|jenkins_public_key_path
+|Jenkins SSH public key path
+|~/.ssh/id_rsa.pub
+|No
```

Comment:
Looks ok. Maybe a bit light on the description but for upstream docs I think this is fine. wdyt @aliok ?

Reply:
@PhilipGough I kept it similar to what was used in others but I will update it if @aliok and you agree I should

Reply:
Yep, absolutely. LGTM @cfoskin